CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
+## 2015-04-08
```

Comment:
Yeah, I *think* so. I like the idea of maintain a changelog, but versioning this seems hard.